src/tabpfn/model/loading.py

```diff
@@ -718,3 +738,126 @@ def save_tabpfn_model(model: nn.Module, save_path: Path | str) -> None:
 
     # Save the checkpoint
     torch.save(checkpoint, save_path)
+
+
+def save_fitted_tabpfn_model(estimator: BaseEstimator, path: Path | str) -> None:
```

Comment:
not sure if it's better but what would you think about instead 1) setting .model_  to None 2) pickling 3) when loading, putting the model back? (basically the same thing you're doing but relying on pickle for everything instead of saving the params yourself).

Reply:
sounds good!